service/src/main/java/bio/terra/profile/service/profile/flight/create/CreateProfileFlight.java

```diff
@@ -32,23 +33,23 @@ public CreateProfileFlight(FlightMap inputParameters, Object applicationContext)
     AuthenticatedUserRequest user =
         inputParameters.get(JobMapKeys.AUTH_USER_INFO.getKeyName(), AuthenticatedUserRequest.class);
 
-    addStep(new GetProfileStep(profileDao, profile));
-    addStep(new CreateProfileStep(profileDao, profile, user));
-    switch (profile.cloudPlatform()) {
-      case GCP:
-        addStep(new CreateProfileVerifyAccountStep(crlService, profile, user));
-        break;
-      case AZURE:
-        addStep(
-            new CreateProfileVerifyDeployedApplicationStep(
-                azureService, profile, azureConfig, user));
-        break;
-    }
-    addStep(new CreateProfileAuthzIamStep(samService, profile, user));
+    MetricUtils.recordProfileCreation(
```

Comment:
This code only sets up the flight, but does not actually execute it. Since we call this code synchronously [here](https://github.com/DataBiosphere/terra-billing-profile-manager/blob/d6ba9f90ad7661dfa387b7b933767bc45c38b433/service/src/main/java/bio/terra/profile/service/profile/ProfileService.java#L60) that might be the place to set the timer.

Reply:
Ah, thanks! I will definitely move it.

Reply:
Yeah, suddenly I have meaningful times! I was stopping a lot in the debugger when I was testing, so I was getting some  events with realistic times and didn't pay attention to the unrealistic times when not stopping at breakpoints.

```
# HELP profile_creation_time_seconds  
# TYPE profile_creation_time_seconds summary
profile_creation_time_seconds_count{cloudPlatform="AZURE",} 2.0
profile_creation_time_seconds_sum{cloudPlatform="AZURE",} 21.369672376
# HELP profile_creation_time_seconds_max  
# TYPE profile_creation_time_seconds_max gauge
profile_creation_time_seconds_max{cloudPlatform="AZURE",} 12.16705965